core/app/models/spree/app_configuration.rb

```diff
@@ -44,6 +44,7 @@ class AppConfiguration < Preferences::Configuration
     preference :company, :boolean, default: false # Request company field for billing and shipping addr
     preference :currency, :string, default: 'USD'
     preference :default_country_id, :integer
+    preference :enable_coupons, :boolean, default: true
```

Comment:
@bbonislawski maybe change this preference name to just `:coupon_codes`  and add a comment describing this option :)

Reply:
@damianlegawiec done